runtime/dart_init.cc

```diff
@@ -194,10 +199,19 @@ Dart_Isolate ServiceIsolateCreateCallback(const char* script_uri,
   return nullptr;
 #else   // FLUTTER_RUNTIME_MODE
   UIDartState* dart_state = new UIDartState(nullptr, nullptr);
+
+  bool is_running_from_kernel = kernel_platform != nullptr;
```

Comment:
Should this be GetKernelPlatformBinary() != nullptr to preserve the abstraction?

Reply:
Done.